graylog2-web-interface/src/views/components/searchbar/saved-search/SavedSearchForm.jsx

```diff
@@ -43,7 +43,7 @@ const SavedSearchForm = (props: Props) => {
                 placement="left"
                 target={target}>
         <Popover title="Name of search" id="saved-search-popover">
-          <form onSubmit={stopEvent}>
+          <form onSubmit={stopEvent} style={{ width: '210px' }}>
```

Comment:
Should we extract this into a styled component instead of using an inline style?